package.json

```diff
@@ -51,6 +51,11 @@
 						"Automatically open terminals, even if there is only one workspace folder."
 					],
 					"description": "Open terminals on start and every time a workspace folder is added to the workspace"
+				},
+				"workspace-terminals.switch_terminals": {
```

Comment:
Since the VS Code settings use camelCase I would like to do the same, so I'd prefer `switchTerminals` .